index.js

```diff
@@ -1,31 +1,66 @@
-/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
-   Complete the below for code reviewers' convenience:
+class Parent {
+  constructor(name, gender) {
+    this.name = name
+    this.gender = gender
+  }
```

Comment:
Please, make all properties common for all inhabitants the properties of the basemost class.